mindmeld/components/_config.py

```diff
@@ -455,6 +455,10 @@
     },
 }
 
+DEFAULT_TOKENIZER_CONFIG = {
+    "patterns": [],
```

Comment:
rename to `whitelist_patterns`

Reply:
Can we name it `allow_patterns` instead? There've recently been suggestions to use allowlist/blocklist instead of whitelist/blacklist to be more racially sensitive

Reply:
Should we include the existing built-in patterns here as part of the config? Or at least a note on where to find them, so it's clear to the developer what is currently being used

Reply:
Done!